fraqbot/Local/randomemoji.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import os
+import random
+import sys
+
+from Legobot.Lego import Lego
+
+
+logger = logging.getLogger(__name__)
+LOCAL_DIR = os.path.join(os.path.abspath(os.path.dirname(__file__)))
+
+if LOCAL_DIR not in sys.path:
+    sys.path.append(LOCAL_DIR)
+
+
+from helpers import utils  # noqa: E402
+
+
+class RandomEmoji(Lego):
+    def __init__(self, baseplate, lock, *args, **kwargs):
+        super().__init__(baseplate, lock, acl=kwargs.get('acl'))
+
+    def listening_for(self, message):
+        text = message.get('text')
+
+        return isinstance(text, str) and text.startswith('!emoji')
+
+    def _get_emoji(self, how_many):
+        maximum_how_many = 20
+        minimum_how_many = 1
```

Comment:
These could be set as class properties and then they wouldn't have to be set every time the _get_emoji method is run